ldapauthenticator/ldapauthenticator.py

```diff
@@ -41,6 +41,27 @@ def _server_port_default(self):
         """
     )
 
+    server_ca_file = Unicode(
+        config=True,
+        help="""
+        Path of the CA certificate file for the *secure* LDAP server
+        """
+    )
+
+    client_certificate_file = Unicode(
+        config=True,
+        help="""
+        Path of the certificate file for the LDAP client
+        """
+    )
+
+    client_key_file = Unicode(
+        config=True,
+        help="""
+        Path of the key file for the LDAP client
+        """
+    )
+
```

Comment:
I'm still not satisfied with the documentation that i wrote, but i think that should be left for another moment.